assignments/arrays.js

```diff
@@ -62,35 +315,76 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // }
 
 // ==== Challenge 1 ====
-// The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+// The dealer can't recall the information for a car with an id of 33 on his
+// lot. Help the dealer find out which car has an id of 33 by logging the car's
+// year, make, and model in the console log provided to you below:
 
+function getCar(inventory, carId) {
+    for (i = 0; i < inventory.length; i++) {
+        if (inventory[i].id == carId) {
+            let car = inventory[i];
+            return `Car ${carId.toString()} is a ${car.car_year.toString()} ${
```

Comment:
Being explicit is good, but you actually don't have to convert your numbers to be strings here (especially since using a template literal is already doing that for you):
```javascript
 `Car ${carId} is a ${car.car_year} ${ ...`
```

Reply:
Sorry my python showed.